indico/modules/events/abstracts/templates/abstract/_common.html

```diff
@@ -33,3 +35,22 @@
         <i class="abstract-comment-visibility {{ visibility.name }} icon-shield" title="{{ visibility.title }}"></i>
     {% endif %}
 {% endmacro %}
+
+{% macro render_instruction_box(instructions, title) %}
+    {% if instructions %}
+        {% set link %}
+            <a data-title="{{ title }}"
+               data-content=".hidden-instructions"
+               data-close-button
+               data-ajax-dialog>
+        {% endset %}
+        {% set endlink='</a>' %}
+        {% set text = caller(link|safe, endlink|safe) %}
```

Comment:
Why do we need this sorcery here? Can't we simply render `<a ...></a>` within `{% call message_box(...) %}`?